tests/terraform/checks/resource/aws/example_SNSTopicPolicyAnyPrincipal/main.tf

```diff
@@ -27,6 +27,37 @@ resource "aws_sns_topic_policy" "sns_tp1" {
 POLICY
 }
 
+# pass
```

Comment:
@ChanochShayner this should be `UNKNOWN`, right?
And now I see that this policy below is invalid - example valid policy where the SourceOwner is valid can be found [here](https://docs.aws.amazon.com/sns/latest/dg/sns-access-policy-use-cases.html#sns-allow-specified-service-to-publish-to-topic)

GJ!